contracts/MyStrategy.sol

```diff
@@ -349,15 +392,16 @@ contract MyStrategy is BaseStrategy, ReentrancyGuardUpgradeable {
                 if (difference > 0) {
                     IWeth(address(WETH)).deposit{value: difference}();
                     nonZeroDiff = true;
-                    _handleRewardTransfer(address(WETH), difference);
+                    _handleRewardTransfer(address(WETH), address(0), difference);
```

Comment:
Is eth always going to be forwarded to the bribe processor?

Reply:
Ended up allowing for redirection using WETH as the token to map to.